src/MongoDB.Driver.Core/Core/Authentication/SaslAuthenticator.cs

```diff
@@ -117,21 +116,24 @@ public async Task AuthenticateAsync(IConnection connection, ConnectionDescriptio
                     {
                         throw CreateException(connection, ex);
                     }
-
```

Comment:
Don't remove all these blank lines.

They serve to separate logically distinct chunks of code.